sansa-query-spark/src/test/scala/net/sansa_stack/query/spark/dof/Logger.scala

```diff
@@ -0,0 +1,13 @@
+package net.sansa_stack.query.spark.dof
+
+import net.sansa_stack.query.spark.dof.bindings.{ Result, ResultRDD }
+
+class Logger(logAllowed: Boolean) extends Serializable {
```

Comment:
In case you will remove all the logs/prints I think there is no need to have such a logger.

Reply:
done